atomic_reactor/schemas/user_params.json

```diff
@@ -31,6 +31,26 @@
     "koji_task_id": {"type": "integer"},
     "koji_upload_dir": {"type": "string"},
     "name": {"type": "string"},
+    "parent_images_digests": {
+      "type": "object",
+      "patternProperties": {
+        "^.+:.+": {
+          "description": "Image with tags",
+          "type": "object",
+          "patternProperties": {
+            "^.+": {
```

Comment:
Is this equivalent to "."?

Reply:
No, it is equivalent to `..*`, i.e. at least one character expected

Reply:
Yes, because I have omitted `$` at the end, so it is equivalent of `.`